packages/testsuite/cypress/e2e/elytron-oidc-client/test-oidc-security-rbac.cy.ts

```diff
@@ -0,0 +1,67 @@
+describe("TESTS: Access secured by Elytron OIDC client with RBAC", () => {
+  let wildfly: string;
+  let ssoServer: string;
+
+  before(() => {
+    cy.startWildflyContainerSecured()
+      .then((result) => {
+        wildfly = result as string;
+      })
+      .then(() => {
+        cy.startSsoContainer().then((result) => {
+          ssoServer = result as string;
+          // the following CLI commands secure the web-console by OIDC, with RBAC enabled
+          cy.executeInWildflyContainer(
```

Comment:
As the other commands depend on this, we should leverage `.then` as the function returns promise and the execution is async.